src/components/forms/ResetPWForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useContext, useState } from "react";
+import { Formik, Form } from "formik";
+import { useHistory } from "react-router-dom";
+import { Alert } from "react-bootstrap";
+import gpib from "../../apis/gpib";
+import Input from "./form-inputs/Input";
+import SubmitSpinnerButton from "./SubmitSpinnerButton";
+import { minPasswordLength } from "../../constants";
+import { AuthContext } from "../Auth";
+
+const validate = ({ password, passwordMatch }) => {
+  const requiredMsg = "This field is required";
+  const errors = {};
+
+  // Required fields
+  if (!password) errors.password = requiredMsg;
+  if (!passwordMatch) errors.passwordMatch = requiredMsg;
+
+  // Formatting
+  if (password.length < minPasswordLength)
+    errors.password = `Password must be at least ${minPasswordLength} characters`;
+
+  // Password match
+  if (password !== passwordMatch)
+    errors.passwordMatch = "Passwords do not match";
+
+  return errors;
+};
+
+const initialValues = {
+  password: "",
+  passwordMatch: ""
+};
+
+const ResetPWForm = ({ onSuccess, onError, email, token }) => {
```

Comment:
I think for things like this we should spell it out "ResetPasswordForm"

Reply:
Good call. I have updated this